src/main/java/com/wavefront/jersey/sdk/ApplicationTags.java

```diff
@@ -0,0 +1,133 @@
+package com.wavefront.jersey.sdk;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Metadata about your application represented as tags in Wavefront.
+ *
+ * @author Sushant Dewan (sushant@wavefront.com).
+ */
+public class ApplicationTags {
```

Comment:
Can class this be in the core sdk as other sdk's will need the same tags?

Reply:
Let's do that as a follow-up change. We can delete this class and fetch it from core-sdk.